src/fontstash.h

```diff
@@ -797,6 +797,17 @@ int fonsAddFallbackFont(FONScontext* stash, int base, int fallback)
 	return 0;
 }
 
+void fonsResetdFallbackFont(FONScontext* stash, int base)
```

Comment:
Remove extra D:
fonsReset*d*FallbackFont

Reply:
Oops! Just fixed the typo. :)